scalableminds/webknossos
# Improve styling of buttons in toolbar & refactor CSS

![image](https://github.com/user-attachments/assets/ecec3a2d-5688-4c94-929c-59f097f768b1)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- use the toolbar and make sure that all buttons look nice in various states (also test disabled buttons, such as the interpolate tool when no interpolation partner exists yet)
- double check that the status bar looks good

### TODOs:
- [x] the save button does not have the primary color. I don't know why, though

### Issues:
- fixes #8261

------
(Please delete unneeded items, merge only when none are left open)
